quinn/src/tests.rs

```diff
@@ -98,11 +98,16 @@ fn local_addr() {
 }
 
 #[test]
-fn read_after_close() {
+fn read_after_close_and_export_keying_material() {
```

Comment:
Should this be two separate tests?

Reply:
Sure, that would work! I'm happy to make the change. I did this to avoid duplicating the setup code, which would otherwise end up being most of the new function.

Would you like me to split it?

Reply:
Yeah; if we end up with duplicate code then it's better to find a way to factor that out than to conflate independent tests.

Reply:
Done. :+1: